shippable.yml

```diff
@@ -0,0 +1,45 @@
+#### Shippable CI CONFIG
+
+language: java
+
+jdk:
+  - oraclejdk8
+
+git:
+  submodules: true
+
+# Avoid duplicate builds for master and the PR branch
+branches:
+  except: master
+
+env:
+  global:
+    - CRATE_TESTS_SQL_REQUEST_TIMEOUT="20"
+    - CRATE_TESTS_NO_IPV6=true
+    - _JAVA_OPTIONS="-Xms1g -Xmx1g"
+    - GRADLE_OPTS="-Dorg.gradle.daemon=false"
+
+# Create and use a non-root user for the build
+# to avoid errors in es submodule.
+build:
+  ci:
+    - adduser --disabled-password --gecos "" ciuser
+    - CIUSER_BUILD_DIR=/home/ciuser/build
+    - mv $SHIPPABLE_BUILD_DIR $CIUSER_BUILD_DIR
+    - chown -R ciuser:ciuser $CIUSER_BUILD_DIR
+    - su -c "export IS_SHIPPABLE_CI_BUILD=true && cd $CIUSER_BUILD_DIR && ./gradlew -s :sql:test" ciuser
+  cache: true
```

Comment:
@mfussenegger This cache configuration doesn't seem to work :-( Still downloads all dependencies.

Reply:
@mfussenegger The issue is with the workaround we do with the `ciuser` to avoid running as root. I can maybe completely remove the cache part.